apptainer/moose-dev.def

```diff
@@ -21,6 +22,10 @@
 
 {#- The installation location for wasp                                                        -#}
 {%- set WASP_DIR = '/opt/wasp' -%}
+{#- The installation location for libtorch                                                    -#}
+{%- set LIBTORCH_DEST = '/opt' -%}
+{#- The version of libtorch to be defined                                                     -#}
+{%- set LIBTORCH_VER = '2.1.0' -%}
```

Comment:
Put this as a variable in `%post` instead, we only use jinja variables like this for things that need to be used in both `%post` and `%env`